### ISSUE TITLE
connect java code to ionic

### ISSUE THREAD

**Issue by shaysegal** [TechnionYP5777/SmartParking#34]
since we work with ionic now we need to connect the java code we wrote, which contain the logic ,to the ionic app.
we'll use REST api.

i'll open issue/wiki that explains the process

**Comment by shaysegal** [comment]
until now :
- read this article : http://www.gajotres.net/best-available-java-restful-micro-frameworks/3/
   - tend to go with jersey https://jersey.java.net/
- read the quick guide of jersey https://jersey.java.net/documentation/latest/getting-started.html
- and saw video about how working with jersey and eclipse https://www.youtube.com/watch?v=qmEVsUJh2d8

TBC

**Comment by Shahar-Y** [comment]
As discussed with @shaysegal, this issue is assigned to @DavidCohen55 and myself. 
We will continue what Shay has begun. 


![i will finish what you started - imgur](https://cloud.githubusercontent.com/assets/23077804/25518090/089d8c40-2bfb-11e7-8724-22c3151ba9e2.gif)

**Comment by shaysegal** [comment]
adding a video of ionic app using REST sevice 
https://www.youtube.com/watch?v=ilM8YorL_jI&t=2654s

**Comment by Shahar-Y** [comment]
Today I kept trying to fix the web services fro Ionic, but I am still getting "Error: No provider for Http" in the login tab, and I'm not sure why.. Changing the browser didn't help either.
 @DavidCohen55 will continue from where I stopped today, Hoping to resolve the problem.

**Comment by Shahar-Y** [comment]
After some serious struggle along with Yael, we managed to establish a connection to the server and present data from it to the ionic app. We can pull data from websites in Json format, including our localhosts. So now we have both client side and server side with a very basic functionality.

Today @DavidCohen55 and I tried to run Zahi's Ionic project, with no success on both computers and for some reason, it failed to run. We worked with him for about an hour and a half, but it didn't help.
We will try to figure it out in the next few days, and then start to work on issue #37.
